hadoop-hdds/docs/content/_index.md

```diff
@@ -34,11 +34,12 @@ ref "JavaApi.md"
 >}}), [S3 protocol support] ({{< ref "S3.md" >}}), and a [command line interface]
 ({{< ref "shell/_index.md" >}})  which makes it easy to use Ozone.
 
-Ozone consists of volumes, buckets, and Keys:
+Ozone consists of volumes, buckets and keys:
 
 * Volumes are similar to user accounts. Only administrators can create or delete volumes.
-* Buckets are similar to directories. A bucket can contain any number of keys,  but buckets cannot contain other buckets.
-* Keys are similar to files. A bucket can contain any number of keys.
+* Buckets are similar to directories. A bucket can contain any number of keys
+ but buckets cannot contain other buckets.
```

Comment:
why such hate against commas?

Reply:
For this one again, I guess my British English is causing the itch since we used a conjunction the comma was bothering me.